package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "reactions",
+  "version": "1.0.0",
+  "description": "Light-weight reactions module",
+  "main": "index.js",
+  "scripts": {
+    "build:dev": "webpack --mode development",
+    "build:watch": "webpack --mode development --watch",
+    "build": "webpack --mode production",
+    "linter": "tslint ./src/**/*.ts --fix"
```

Тут лучше заюзать глагол

```suggestion
    "lint": "tslint ./src/**/*.ts --fix"
```